### ISSUE TITLE
In quest Kyle's Gone Missing! in Mulgore,the npc not gives to you quest item for call the dog in the bridge by bloodhoof village (I'm sorry this quest nots is bugged, its is my error lol sorry) Now the dog is digging lol hehe.Seriously, this is the best server.

### ISSUE THREAD

**Issue by Veesc** [dalaranwow/dalaran-wow#4823]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)

**Expected behaviour**: Tell us what should happen instead

**Steps to reproduce the problem**:

1. 
2. 
3. 

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Crispybuilt** [comment]
This is not how you form a bug report.

**Comment by Rushor** [comment]
Best title ever :D 

You understood something wrong here. The quest is https://wowgaming.altervista.org/aowow/?quest=11129.

Here is the meat that u need: Tender Strider Meat (https://wowgaming.altervista.org/aowow/?item=33009) drop from Adult Plainstrider (https://wowgaming.altervista.org/aowow/?npc=2956) or Elder Plainstrider (https://wowgaming.altervista.org/aowow/?npc=2957)

You need to drop this item in front of Kyle, then is going to eat it/dance/fullfill your questrequirement. 

Thanks, i will close the issue then.
